File: 프로그래머스/unrated/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    lux, luy, rdx, rdy = -1, 1e9, 0, 0
    for i in range(len(wallpaper)):
        if "."*len(wallpaper[0]) != wallpaper[i]:
            if lux == -1:
                lux = i
            rdx = i
        for j in range(len(wallpaper[0])):
            if wallpaper[i][j] == "#":
                luy = min(luy, j)
                rdy = max(rdy, j)
    
    return [lux, luy, rdx+1, rdy+1]

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int lux = -1;
        int luy = wallpaper[0].length();
        int rdx = 0;
        int rdy = 0;

        for (int i=0; i<wallpaper.length; i++) {
            if (wallpaper[i].contains("#")) {
                if (lux == -1) {
                    lux = i;
                }
                rdx = i;
            }

            for (int j=0; j<wallpaper[i].length(); j++) {
                if (wallpaper[i].charAt(j) == '#') {
                    luy = Math.min(luy, j);
                    rdy = Math.max(rdy, j);
                }
            }
        }

        return new int[]{lux, luy, rdx+1, rdy+1};
    }
}